modules/administration/pages/kubernetes.adoc

```diff
@@ -47,25 +45,24 @@ _Kubernetes_ clusters are registered with SUSE Manager as ``virtual host manager
 Registration and authorization begins with importing a `kubeconfig` file using Kubernetes official command line tool ``kubectl``.
 
 .Procedure: Registering a Kubernetes Cluster with {productname}
-. Select menu:Systems[Virtual Host Managers] from the navigation menu.
-. Expand the `Create` dropdown in the upper right corner of the page and select menu:Kubernetes Cluster[] .
-. Input a label for the new Virtual Host Manager.
-. Select the `kubeconfig` file which contains the required data for the Kubernetes cluster.
+. Select menu:Systems[Virtual Host Managers] from the main navigation menu of the {webui}.
```

Comment:
I've been using the phrasing "In the {productname} {webui}, navigate to menu:foo[bar].

Reply:
thanke!